content-plugins.md

```diff
@@ -0,0 +1,245 @@
+                    <li>Version content and rollback if necessary.</li>
+                    <li>Support for the new multi-role content type from Galaxy.</li>
+                </ul>
+            </div>
+            <div class="tab-pane fade" id="deb">
+                <p>Manage Debian content</p>
+                <ul>
+                    <li>Synchronize remote repository content and metadata locally.</li>
+                    <li>Upload your own content.</li>
+                    <li>Publish content to one or more repositories.</li>
+                  </ul>
+            </div>
+            <div class="tab-pane fade" id="python">
+                <p>Manage Python content</p>
+                <ul>
+                    <li>Synchronize Python packages from PyPI locally.</li>
+                    <li>Upload your own Python packages.</li>
+                    <li>Publish and organize Python packages just like PyPI.</li>
+                    <li>Install Python packages using pip as published by Pulp.</li>
+                </ul>
+            </div>
+            <div class="tab-pane fade" id="gem">
+                <p>Manage RubyGem content</p>
+                <ul>
+                    <li>Synchronize remote repository content and metadata locally.</li>
+                    <li>Upload your own content.</li>
+                    <li>Publish content to one or more repositories.</li>
+                  </ul>
+            </div>
+            <div class="tab-pane fade" id="maven">
+                <p>Manage Maven content</p>
+                <ul>
+                <li>Synchronize packages from a remote to local repository.</li>
+                <li>Upload your own Maven content.</li>
+                <li>Publish and organize packages.</li>
+                  </ul>
+            </div>
+            <div class="tab-pane fade" id="cookbook">
+                <p>Manage Chef Cookbook content</p>
+                <ul>
+                    <li>Sync Cookbook content from a remote to local repository.</li>
+                    <li>Upload your own content.</li>
+                    <li>Publish and organize content.</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</div>
+
+## Pulp 3 Content Plugins Information
+
+This table contains links to information and sources for all Pulp 3 content plugins. If a plugin is missing [contact us](https://www.redhat.com/mailman/listinfo/pulp-list).
+
+| Pulp Plugin | Docs | Source | Tracker | Install with PyPI | Install with RPM |
+|-------|--------|---------|--------|---------|-------- |--------- |
+| Ansible plugin | <a href="https://github.com/pulp/pulp_ansible/blob/master/README.rst">Ansible plugin docs</a> | <a href="https://github.com/pulp/pulp_ansible">Ansible plugin source</a> | <a href="https://pulp.plan.io/projects/ansible_plugin?jump=welcome">Ansible plugin tracker</a> | <a href="https://pypi.org/project/pulp-ansible/">Yes</a> | No |
```

Comment:
Note to self: replace this doc link with: https://pulp-ansible.readthedocs.io/en/latest/